packages/spec/integration/graphql-no-ssr/mock-schema.js

```diff
@@ -0,0 +1,34 @@
+import { makeExecutableSchema } from '@graphql-tools/schema';
+
+const typeDefs = `
+  type Subject {
+    id: ID!
+    title: String
+  }
+
+  type Query {
+    subject: Subject
+  }
+`;
+
+const values = Array.from({ length: 10 }, () => ({
```

Comment:
Why do we need the `Server` value 10 times? Couldn't we just write it out?

Reply:
For debugging purposes when you run this app and check it in the browser it's less annnoying, when you can reload the page more than once, before an error is thrown because of the GraphQL API not responding with a string value. 🤷‍♂️